src/screening.cpp

```diff
@@ -664,24 +912,9 @@ int main(int nargs, char **argv) {
 		int count = model_reservoirs(search_config.grid_square, pour_points, flow_directions, DEM_filled, flow_accumulation, filter);
 		search_config.logger.debug("Found " + to_string(count) + " reservoirs. Runtime: " + to_string(1.0e-6*(walltime_usec() - t_usec)) + " sec");
 		printf(convert_string("Screening finished for "+search_config.search_type.prefix()+str(search_config.grid_square)+". Runtime: %.2f sec\n"), 1.0e-6*(walltime_usec() - start_usec) );
-  } else {
-	// Depression volume finding for pits
-	if (search_config.search_type == SearchType::BULK_PIT) {
-		t_usec = walltime_usec();
-		Model<short> *DEM = read_DEM_with_borders(search_config.grid_square, border);
-		Model<double>* DEM_filled_no_flat = fill(DEM);
-		Model<short>* DEM_filled = new Model<short>(DEM->nrows(), DEM->ncols(), MODEL_SET_ZERO);
-		DEM_filled->set_geodata(DEM->get_geodata());
-		for(int row = 0; row<DEM->nrows();row++) {
-			for(int col = 0; col<DEM->ncols();col++) {
-				DEM_filled->set(row, col, convert_to_int(DEM_filled_no_flat->get(row, col)));
-			}
-		}
-
-		depression_volume_finding(DEM_filled);
-		printf(convert_string("Volume finding finished for "+str(search_config.grid_square)+". Runtime: %.2f sec\n"), 1.0e-6*(walltime_usec() - t_usec) );
-	}
-
+   
+   // Deprecated Brownfield searching based on individual pit Shapefiles
```

Comment:
Do we definitely want to deprecate this? It was a much more accurate search as it took a list of height-volume pairs and would pick the best height to fill the pit to (at least delete shapefiles as I don't believe there was a shapefile input here)

Reply:
Will add the volume-altitude pairs into the new algorithm again (oversight by me) - old method was less accurate at actually finding the pit shapes, but agree that we should still be able to have partially filled pits for pairing.

Reply:
I've now developed the volume-altitude pair functionality for the new bulk_pits algorithm, so we can estimate the best height to fill the pit again as part of pairing